src/client/src/containers/SortablePageList/index.tsx

```diff
@@ -118,7 +116,7 @@ const SortablePageList = (props: Props) => {
           <div className={styles.dropdownTitle}>
             {`${props.intl!.formatMessage(messages.pages)} (${
               pages.length >= 0 ? pages.length : ""
-            })`}
+              })`}
```

Comment:
whitespace change. Looks like the person responsible for this branch doesn't have the same automatic linter/formatter installed than the rest of the team